content/en/dora_metrics/deployments/_index.md

```diff
@@ -118,13 +117,13 @@ There are two requirements for calculating Change Lead Time:
 
 ### Breakdown metrics
```

Comment:
should we move Breakdown metrics to content/en/dora_metrics/data_collected/_index.md?

Reply:
I agree!

Reply:
Will do this in a follow up commit.